packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -56,17 +58,25 @@ def _estimate(operator: Estimatable, state: ITensorStateT) -> Estimate[complex]:
     op = convert_operator(operator, s)
 
     # calculate expectation value
-    psi = jl.apply(circuit, psi)
+    psi = jl.apply(circuit, psi, **kwargs)
     exp: float = jl.expectation(psi, op)
 
     return _Estimate(value=exp, error=0.0)
 
 
-def create_itensor_mps_estimator() -> QuantumEstimator[ITensorStateT]:
+def create_itensor_mps_estimator(**kwargs: Any) -> QuantumEstimator[ITensorStateT]:
```

Comment:
I think it's better if some commonly used arguments are given explicitly so that IDEs can provide the code suggestions.

```suggestion
def create_itensor_mps_estimator(
    mindim: Optional[int] = None,
    maxdim: Optional[int] = None,
    cutoff: Optional[float] = None,
    **kwargs: Any,
) -> QuantumEstimator[ITensorStateT]:
```